android/src/app/organicmaps/widget/menu/NavMenu.java

```diff
@@ -208,11 +208,49 @@ private void updateSpeedView(@NonNull RoutingInfo info)
 
     Pair<String, String> speedAndUnits = StringUtils.nativeFormatSpeedAndUnits(last.getSpeed());
 
+    // (temporarily) show the speed limit together with the current speed.
+    String speedIndicator = formatSpeed(speedAndUnits, info);
+    mSpeedValue.setText(speedIndicator);
+
     mSpeedUnits.setText(speedAndUnits.second);
-    mSpeedValue.setText(speedAndUnits.first);
     mSpeedViewContainer.setActivated(info.isSpeedLimitExceeded());
   }
 
+  private String formatSpeed(Pair<String, String> speedAndUnits, RoutingInfo info)
+  {
+    double currentSpeedMps = Double.parseDouble(speedAndUnits.first);
+    String speedIndicator = String.format("%.0f", currentSpeedMps);
+
+    if (info.speedLimit > 0.0)
+    {
+      double speedLimit = convertMpsIntoUnits(info.speedLimit, speedAndUnits.second);
+      speedIndicator += String.format("/%.0f", speedLimit);
```

Comment:
[Pure suggestion as I would love to see this PR merged even without it and I wouldn't slow down the reviewing process]

iOS [pr](https://github.com/organicmaps/organicmaps/pull/2963/files#diff-689d1838cea59440095256c64d22da973383adbb1143f45a1c545d32621b1de4R160) use space around the slash:
```suggestion
      speedIndicator += String.format(" / %.0f", speedLimit);
```

Reply:
> iOS use space around the slash:

The initial implementation was like that. Unfortunately, it takes too much screen space, in my opinion. Hence I narrowed it down.

Reply:
Oh ! ok. Sorry I didn't see this from the history because of the force push.

Is there anything I can do to help the merge of this pr ?
Speed limit information is one of key features I really love to see in Organic Maps